### PR TITLE
HTML-CSS-Popup Task

### PR DESCRIPTION
**Pull request by EnF1NeeT**
### HTML-CSS-Popup

[Demo](https://enf1neet.github.io/enf1neet_popup_demo/)
[Code base](https://github.com/EnF1NeeT/enf1neet_popup_demo)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by EnF1NeeT** [comment]
Update index.html and style.css 15/08/222

**Comment by EnF1NeeT** [comment]
Hi! Can I get feedback on my latest commit and in general on PR? :)

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Alumni+Sans:ital,wght@0,300;1,300&display=swap" rel="stylesheet">
+    <title>Kottanspost</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul class="nav-list list">
+          <li class="nav-item">
+            <input type="checkbox" id="menu-check" class="input-check">
+            <label for="menu-check" class="check"  style="cursor: pointer;">Menu</label>
```
Comment (A-Ostrovnyy) [inline submissions/EnF1NeeT/HTML-CSS_popup/index.html:16]: Avoid inline styles, use CSS classes for it.

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            <div class="main-menu">
+              <ul class="popup-grid list">
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Mail.svg" alt="Mail" class="icon"> Mail</a></li>
```
Comment (A-Ostrovnyy) [inline submissions/EnF1NeeT/HTML-CSS_popup/index.html:19]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/payments.svg" alt="Payments" class="icon"> Payments</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/store.svg" alt="Store" class="icon"> Store</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Settings.svg" alt="Settings" class="icon"> Settings</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/logout.svg" alt="Logout" class="icon"> Logout</a></li>
+                </ul>
+              </div>
+            </div>
+          </li>
+          <li class="logo"><h1>Kottanspost</h1></li>
+          <li class="greeting"><h2>Hi, Kottan!</h2><img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt=""></li>
```
Comment (A-Ostrovnyy) [inline submissions/EnF1NeeT/HTML-CSS_popup/index.html:50]: Fix vertical alignment for a better view
image

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/security.svg" alt="Security" class="icon"> Security</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/payments.svg" alt="Payments" class="icon"> Payments</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/store.svg" alt="Store" class="icon"> Store</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Settings.svg" alt="Settings" class="icon"> Settings</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/logout.svg" alt="Logout" class="icon"> Logout</a></li>
+                </ul>
+              </div>
+            </div>
+          </li>
+          <li class="logo"><h1>Kottanspost</h1></li>
```
Comment (A-Ostrovnyy) [inline submissions/EnF1NeeT/HTML-CSS_popup/index.html:49]: The logo should be a link and lead users to the home page. You can put a link inside `h1`

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+          <li class="greeting"><h2>Hi, Kottan!</h2><img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt=""></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="posts">
+        <div class="post">
```
Comment (A-Ostrovnyy) [inline submissions/EnF1NeeT/HTML-CSS_popup/index.html:56]: For every post better to use the tag `article` for better semantics.

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+          <div class="topzone">
+            <img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt="">
+            <p>@Jonnie</p>
+          </div>
+          <div class="post-img">
+            <img src="./img/post1.jpg" class="post-img" alt="">
```
Comment (A-Ostrovnyy) [inline submissions/EnF1NeeT/HTML-CSS_popup/index.html:62]: This is a content image and `alt` must be populated.

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+              Like
+            </div>
+            <div class="btn">
+              <img src="./img/forum_FILL0_wght400_GRAD0_opsz48.png" alt="comment">
+              Comment
+            </div>
+            <div class="btn">
+              <img src="./img/send_FILL0_wght400_GRAD0_opsz48.png" alt="share">
+              Share
+            </div>
```
Comment (A-Ostrovnyy) [inline submissions/EnF1NeeT/HTML-CSS_popup/index.html:76]: Why you don't use `button` tag for this element?

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+          </div>
+          <div class="post-time">posted 5h ago</div>
```
Comment (A-Ostrovnyy) [inline submissions/EnF1NeeT/HTML-CSS_popup/index.html:78]: For date and time use HTML tag `time`

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+        </div>
+      </div>
+    </main>
+  </body>
+</html>
```
Comment (A-Ostrovnyy) [inline submissions/EnF1NeeT/HTML-CSS_popup/index.html:155]: Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+            <p>@Jonnie</p>
+          </div>
+          <div class="post-img">
+            <img src="./img/post3.jpg" class="post-img" alt="">
+          </div>
+          <div class="action-zone">
+            <div class="btn">
+              <img src="./img/favorite_FILL0_wght400_GRAD0_opsz48.png" alt="like">
+              Like
+            </div>
```
Comment (A-Ostrovnyy) [inline submissions/EnF1NeeT/HTML-CSS_popup/index.html:116]: Add `cursor: pointer` to all interactive elements.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Awesome design!
See my comments below.

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 1 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 2 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 3 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html:50]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 4 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 5 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 6 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html:62]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 7 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 8 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 9 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 10 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/payments.svg" alt="" class="icon"> Payments</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/store.svg" alt="" class="icon"> Store</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Settings.svg" alt="" class="icon"> Settings</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/logout.svg" alt="" class="icon"> Logout</a></li>
+                </ul>
+              </div>
+            </div>
+          </li>
+          <li class="logo"><a href="./index.html" class="logo"><h1>Kottanspost</h1></a></li>
+          <li class="greeting"><h2>Hi, Kottan!</h2><img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt=""></li>
```
Comment (A-Ostrovnyy) [inline submissions/EnF1NeeT/HTML-CSS_popup/index.html:50]: Avoid morphing images, please fix the aspect ratio of the icon 
image

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+              Comment
+            </button>
+            <button class="btn">
+              <img src="./img/send_FILL0_wght400_GRAD0_opsz48.png" alt="share" class="btn-img">
+              Share
+            </button>
+          </div>
+          <div class="post-time">posted <time datetime="24.5.3222">posted 5h ago</time></div>
+        </article>
+        <article class="post">
```
Comment (A-Ostrovnyy) [inline submissions/EnF1NeeT/HTML-CSS_popup/index.html:80]: Adding hover on the whole card is confusing to users, they think the whole card is clickable, but in your case, it's not. Also, avoid putting one interactive element into another: invalid markup, the browser doesn't know click on which element should be processed.
image

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+          <div class="topzone">
+            <img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt="post-img">
+            <p>@Jonnie</p>
+          </div>
+          <div class="post-img">
+            <img src="./img/post2.jpg" class="post-img" alt="">
+          </div>
+          <div class="action-zone">
+            <button class="btn">
+              <img src="./img/favorite_FILL0_wght400_GRAD0_opsz48.png" alt="like" class="btn-img">
```
Comment (A-Ostrovnyy) [inline submissions/EnF1NeeT/HTML-CSS_popup/index.html:90]: Avoid moving other elements on hover. Unnecessary rerenders are bad for page performance.

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 14]
```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```
Comment (A-Ostrovnyy) [inline submissions/EnF1NeeT/HTML-CSS_popup/index.html:1]: Don't forget about the mobile version, right now it's hard to read page content.
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 6 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html [hunk 15]
```diff
@@ -0,0 +1,155 @@
...
+          <div class="topzone">
+            <img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt="post-img">
+            <p>@Jonnie</p>
+          </div>
+          <div class="post-img">
+            <img src="./img/post1.jpg" class="post-img" alt="">
+          </div>
+          <div class="action-zone">
+            <button class="btn">
+              <img src="./img/favorite_FILL0_wght400_GRAD0_opsz48.png" alt="like" class="btn-img">
```
[the remark this answers is not in the document]
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 11 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 12 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html:82]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 13 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html]: Fixed in the last commit

File: submissions/EnF1NeeT/HTML-CSS_popup/index.html (hunk 14 above)
Comment (EnF1NeeT) [reply submissions/EnF1NeeT/HTML-CSS_popup/index.html:1]: Fixed in the last commit

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**